doc/chomp_interface/chomp_interface_tutorial.rst

```diff
@@ -49,10 +49,82 @@ Using CHOMP with Your Robot
       <arg name="debug" value="$(arg debug)"/>
       <arg name="planner" value="chomp" />
     </include>
-#. Open the ``move_group.launch`` file in your ``<robot_moveit_config>/launch/`` folder and make two changes. First, add ``<arg name="planner" default="ompl" />`` just under the ``<launch>`` tag and second, within the ``<include ns="move_group">`` tag replace ``<arg name="pipeline" value="ompl" />`` with ``<arg name="pipeline" value="$(arg planner)" />``.
+#. Open the ``move_group.launch`` file in your ``<robot_moveit_config>/launch/`` folder and make two changes.
+
+ * First, add ``<arg name="planner" default="ompl" />`` just under the ``<launch>`` tag, and,
+
+ * Second, within the ``<include ns="move_group">`` tag replace ``<arg name="pipeline" value="ompl" />`` with ``<arg name="pipeline" value="$(arg planner)" />``.
 
 Running the Demo
 ----------------
 If you have the ``panda_moveit_config`` from the `ros-planning/panda_moveit_config <https://github.com/ros-planning/panda_moveit_config>`_ repository you should be able to simply run the demo: ::
 
-  roslaunch panda_moveit_config demo_chomp.launch
\ No newline at end of file
+  roslaunch panda_moveit_config demo_chomp.launch
+
+Testing CHOMP with Obstacles in the Scene
++++++++++++++++++++++++++++++++++++++++++
+To test CHOMP in an evironment with obstacles, you can run any of the sample python scripts:
+
+- `collision_scene_example_cluttered.py <./scripts/collision_scene_example_cluttered.py>`_ or
+
+- `collision_scene_example_sparse.py <./scripts/collision_scene_example_sparse.py>`_.
+
+The first scripts creates a cluttered scene with four ostacles. The second script creates a simple scene with one obstacle. One can change the position/size of the obstacles to change the scene. 
+
+To run the CHOMP planner with obstacles, open two shells. In the first shell start RViz and wait for everything to finish loading: ::
+
+  roslaunch panda_moveit_config demo_chomp.launch
+
+In the second shell, run either of the two commands: ::
+
+  rosrun moveit_tutorials collision_scene_example_cluttered.py
+
+or: ::
+
+  rosrun moveit_tutorials collision_scene_example_sparse.py
+
```

Comment:
Add instructions telling the user to select CHOMP in the MotionPlanning pannel under the Context tab.
You will also want to tell them to move the endeffector around with the imarker and hit plan